src/chrome-api/index.js

```diff
@@ -18,6 +18,15 @@ export function create() {
 var ChromeManager = {
 
     /**
+     * Install plugin
+     * @param {Object} plugin
+     * @param {Function} plugin.install
+     */
+    install: function (plugin) {
```

Comment:
I think, `install` is too general name. It will be better if name be something like `installPlugin` or `registerPlugin`.